projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/defense/protection/dragonborn.json

```
@@ -0,0 +1,18 @@
+{
+  "modifier_id": "tconstruct:dragonborn",
+  "text": [
+    {
+      "text": "Dovah ko krif dilos ahrk qah."
+    },
+    {
+      "text": "工匠，你不会龙语吗？他们最近都在教些什么啊。这是在说，飞翔于天际的龙是致命的，也是受到保护的。",
```

```suggestion
      "text": "工匠，你不会龙语吗？他们最近都在教些什么啊。这句话是说，飞翔于天际的龙十分致命，并且防御力极高。",
```
根据效果简单更改描述。有这个属性你就化身成龙。